doc/guide/build.hpp

```diff
@@ -140,8 +138,8 @@ source as apt installs an older version. So you need to omit
 On Fedora, Red Hat, or CentOS, these same dependencies can be obtained via dnf:
 
 @code
-# dnf install boost-devel boost-math armadillo-devel binutils-devel
-  python3-Cython python3-setuptools python3-numpy python3-pandas ensmallen-devel
+# dnf install armadillo-devel binutils-devel python3-Cython python3-setuptools
+    python3-numpy python3-pandas ensmallen-devel
```

Comment:
Is the extra space on purpose?

Reply:
Nope, I added it accidentally.  Fixed in 9910d24. :+1: